cmd/operator-sdk/test/local.go

```diff
@@ -68,7 +69,8 @@ func newTestLocalCmd() *cobra.Command {
 	testCmd.Flags().StringVar(&tlConfig.namespacedManPath, "namespaced-manifest", "", "Path to manifest for per-test, namespaced resources (e.g. RBAC and Operator manifest)")
 	testCmd.Flags().StringVar(&tlConfig.goTestFlags, "go-test-flags", "", "Additional flags to pass to go test")
 	testCmd.Flags().StringVar(&tlConfig.moleculeTestFlags, "molecule-test-flags", "", "Additional flags to pass to molecule test")
-	testCmd.Flags().StringVar(&tlConfig.namespace, "namespace", "", "If non-empty, single namespace to run tests in")
+	testCmd.Flags().StringVar(&tlConfig.deployNamespace, "deploy-namespace", "", "If non-empty, single namespace to run tests in (deploys namespaced resources here); default: \"\" which resolves to namespace from kubeconfig")
```

Comment:
```suggestion
	testCmd.Flags().StringVar(&tlConfig.deployNamespace, "deploy-namespace", "", "Namespace where the tests will be executed (By default it will be in the default namespace defined in the kubeconfig)")
```